app/assets/javascripts/locationForm.js

```diff
@@ -58,6 +58,13 @@ function addGrid(map)
    _renderCells: function(bounds) {
      var cells = this._cellsInBounds(bounds);
      this.focused_cell_id = cells.length == 0 ? null : cells[0].id;
+     window.gridSquareNWCorner = function(){
```

Comment:
I don't know the use of this function yet. It was just written to create one function. That's why I made it global.